llvm/test/CodeGen/AMDGPU/attr-amdgpu-num-work-groups.ll

```diff
@@ -0,0 +1,84 @@
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 < %s | FileCheck %s
+
```

Comment:
Missing a test for the various IR attribute parsing error conditions

Reply:
Created a new test file to test various errors.